x-pack/test/functional/apps/observability_logs_explorer/flyout.ts

```diff
@@ -79,7 +79,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await dataGrid.clickRowToggle({ columnIndex: 4 });
       await testSubjects.existOrFail('logsExplorerFlyoutLogLevel');
       await dataGrid.closeFlyout();
+    });
 
+    it('should not display a log level badge when not available', async () => {
```

Comment:
For some reason this test (and the others below) were failing because the web elements had become "stale", and were throwing errors. I'm not sure how the changes in this PR would affect this, but breaking these into separate `it()` statements seems to fix the issue and still test the same behavior.

Reply:
[This PR](https://github.com/elastic/kibana/pull/180262) will actually remove most of these tests after moving the doc view implementation, I'll take care of the conflicts but this LGTM.